Removing multiarg infix syntax warnings introduced with Scala 2.13.3

margussipria opened the pull request:
Fixes issue #530

eed3si9n commented:
Thanks for the contribution!

eed3si9n commented:
Not sure which one is causing it but the following error looks suspicious

```scala
tmp/xmlprotocol.scala:584: error: illegal start of simple expression
      (any(List(x,  , =, >,  , L, i, s, t, (, N, o, n, e, ),  , c, o, n, t, a, i, n, s,  , x, ., n, a, m, e, s, p, a, c, e))) ~
                    ^
tmp/xmlprotocol.scala:623: error: ')' expected but '}' found.
  }
  ^
tmp/xmlprotocol.scala:584: error: illegal start of simple expression
      (any(List(x,  , =, >,  , L, i, s, t, (, N, o, n, e, ),  , c, o, n, t, a, i, n, s,  , x, ., n, a, m, e, s, p, a, c, e))) ~
                    ^
tmp/xmlprotocol.scala:623: error: ')' expected but '}' found.
  }
  ^
[info] GeneralTest
[info] ! general.scala file must compile together with GeneralUsage.scala
[error]  RuntimeException: : Error compiling: tmp/GeneralUsage.scala,tmp/general.scala,tmp/xmlmime.scala,tmp/general_import.scala,tmp/xmlprotocol.scala,tmp/scalaxb.scala  (CompilerMatcher.scala:99)
[error] CompilerMatcher$$anon$2.apply(CompilerMatcher.scala:99)
[error] GeneralTest$$anonfun$1.apply(GeneralTest.scala:29)
[error] GeneralTest$$anonfun$1.apply(GeneralTest.scala:29)
[info] 
[info] ! general.scala file must compile together with CustomizationUsage.scala
[error]  RuntimeException: : Error compiling: tmp/CustomizationUsage.scala,tmp/general.scala,tmp/xmlmime.scala,tmp/general_import.scala,tmp/xmlprotocol.scala,tmp/scalaxb.scala  (CompilerMatcher.scala:99)
[error] CompilerMatcher$$anon$2.apply(CompilerMatcher.scala:99)
[error] GeneralTest$$anonfun$2.apply(GeneralTest.scala:35)
[error] GeneralTest$$anonfun$2.apply(GeneralTest.scala:35)
```

margussipria commented:
Yep, there where 2 cases where format calls where done with "tuple of one" and then running mkString before using result as argument for format method. When I added just simple dot it made first format call and then mkString call for string. 

Added extra parenthesis and now I did manage to run tests locally also and didn't have to wait travis for report
